frontend/packages/knative-plugin/integration-tests/features/eventing/eventing-create-sink-broker.feature

```diff
@@ -0,0 +1,73 @@
+@knative-eventing
+Feature: Knative Eventing Broker Support
+              As a user, I should be able to create broker and sink it with event sources
+
+
+        Background:
+            Given user has installed OpenShift Serverless Operator
+              And user has created Knative Eventing CR
+              And user has created or selected namespace "aut-eventing-broker"
+
+
+        @regression @to-do @odc-5030
+        Scenario: Create Broker form: KC-01-TC01
+            Given user is at create broker form
+             Then user will see Form view default selected
+              And user will see YAML view radio button
+              And user will Application drop down
+              And user will see Name field for broker
+        
+
+        @smoke @to-do @odc-5030
+        Scenario: Create Broker using Form view: KC-01-TC02
+            Given user is at Add page
+             When user selects on Broker from Serverless card
+              And user selects Form view
```

Comment:
By default, form view will be selected

Reply:
Updated as 
```suggestion
              And user selects "Form view"
```

Reply:
form view will be pre-selected by default, user dont have to select "Form view" again. should we remove this?

Reply:
I have added that statement for safety.